recipes/minizip/all/conanfile.py

```diff
@@ -1,8 +1,9 @@
-from conans import ConanFile, tools, CMake
+from conan import ConanFile
+from conan.tools import files
+from conans import CMake
 import functools
-import os
 
-required_conan_version = ">=1.33.0"
+required_conan_version = ">=1.41.0"
```

Comment:
```suggestion
required_conan_version = ">=1.46.0"
```
To use new tools.files